a.core_concepts.md

```diff
@@ -226,12 +223,6 @@ kubectl run busybox --image=busybox --env="NGINX_IP=$NGINX_IP" --rm -it --restar
 
 ```bash
 kubectl get po nginx -o yaml
-# or
```

Comment:
All valid, but I bet the vast majority of people are really just after `-o yaml`

Reply:
The point of having these is for people to see that those are valid ways to do the same thing. Still, it's 6 extra lines.... Maybe in a comment in a single line?